src/azure-cli/azure/cli/command_modules/acs/_loadbalancer.py

```diff
@@ -0,0 +1,101 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from distutils.version import StrictVersion  # pylint: disable=no-name-in-module,import-error
+
+# pylint: disable=no-name-in-module,import-error
+from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfile
+from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfileManagedOutboundIPs
+from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfileOutboundIPPrefixes
+from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterLoadBalancerProfileOutboundIPs
+from azure.mgmt.containerservice.v2019_11_01.models import ResourceReference
+
+
+def set_load_balancer_sku(sku, kubernetes_version):
+    if sku:
+        return sku
+    if kubernetes_version and StrictVersion(kubernetes_version) < StrictVersion("1.13.0"):
+        print('Setting load_balancer_sku to basic as it is not specified and kubernetes \
```

Comment:
pls not use print, but logger.warning or error, eg. https://github.com/Azure/azure-cli/blob/8adf97355a605d147afc26f6c1b6f6afb6651c2f/src/azure-cli/azure/cli/command_modules/cognitiveservices/custom.py#L45

Reply:
@yungezz happy to update this - certainly good feedback, especially for a team like ours that doesn't use python on a daily basis. 

However, I have only moved this code - didn't actually write it - so to scope my PR I would prefer to leave this change for a separate PR.

Please let me know what you think, if you feel strongly about it I can modify it but please advise on the best way to share an "info" (e.g. is there a logger.info) -- in this case it isn't a error or warning it is just a fact that they should be aware of given the rest of their configuration